pkg/config/config_test.go

```diff
@@ -412,3 +409,16 @@ func Test_checkFilePerms(t *testing.T) {
 		})
 	}
 }
+
+func TestConfig_FilterOff(t *testing.T) {
```

Comment:
Just my opinion: but I think the function is simple enough that we don't really need to add this test here.